### PR TITLE
Chakra rework to use mods

### PR DESCRIPTION
**Pull request by nfaletra**
- Gutted hardcoded item id checks in Chakra ability script
- Added mod for VIT mult
- Added mod for status effect removal (mask based)
- Did not update cooldown or merits to current retail values (3 min cooldown and 6 seconds per merit I think?)
- Math not updated to take into account the "function of the player's level" (This was updated in August 2017. There is literally no info about how the level affects recovered HP)

### CODE REVIEW

**Comment by nfaletra** [comment]
Rebased CAUSE TEO!

File: scripts/globals/abilities/chakra.lua [hunk 1]
```diff
@@ -8,48 +8,38 @@
...
-    if (player:hasStatusEffect(dsp.effect.BLINDNESS)) then
-        player:delStatusEffect(dsp.effect.BLINDNESS)
-    end
-
-    if ((body == 12639) or (body == 14474)) then -- Temple Cyclas (+1) equipped
-        if (player:hasStatusEffect(dsp.effect.PARALYSIS)) then
-            player:delStatusEffect(dsp.effect.PARALYSIS)
+function onUseAbility(player, target, ability)
+    local chakraRemoval = player:getMod(dsp.mod.CHAKRA_REMOVAL)
+    if chakraRemoval then
```
Comment (takhlaq) [inline scripts/globals/abilities/chakra.lua]: ~= 0, probably

File: scripts/globals/abilities/chakra.lua [hunk 2]
```diff
@@ -8,48 +8,38 @@
...
-        multi = multi + 0.6
-    end
-
-    local recover = (multi * vit)
-    player:setHP((hp + recover))
+    local recover = player:getStat(dsp.mod.VIT) * (2 + player:getMod(dsp.mod.CHAKRA_MULT) / 10) -- TODO: Figure out "function of level" addition (August 2017 update)
+    player:setHP(player:getHP() + recover)
 
+    local merits = player:getMerit(dsp.merit.INVIGORATE)
     if (merits >= 1) then
```
Comment (takhlaq) [inline scripts/globals/abilities/chakra.lua]: can get rid of these parentheses too

File: scripts/globals/abilities/chakra.lua (hunk 1 above)
Comment (nfaletra) [reply scripts/globals/abilities/chakra.lua]: Still want it to remove Poison and Blindness.

Comment (nfaletra) [reply scripts/globals/abilities/chakra.lua]: Probably doesn't need to be checked at all, actually.

Comment (nfaletra) [reply scripts/globals/abilities/chakra.lua]: Actually we want it to run if it's 0 for Blindness and Poison removal. So check can probably go away.

**Pull request merged**